package.json

```diff
@@ -24,20 +28,41 @@
     "format-all": "npm run prettier-all && npm run eslint-all"
   },
   "homepage": "https://miguelcobain.github.io/ember-leaflet/",
+  "dependencies": {
+    "@babel/core": "^7.25.2",
+    "@embroider/util": "^1.13.2",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "broccoli-funnel": "^3.0.8",
+    "broccoli-merge-trees": "^4.2.0",
+    "ember-auto-import": "^2.10.0",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-htmlbars": "^6.3.0",
+    "ember-composability-tools": "^1.2.0",
+    "ember-in-element-polyfill": "^1.0.0",
+    "ember-render-helpers": "^0.2.0",
+    "fastboot-transform": "^0.1.3",
+    "leaflet": "^1.9.2",
```

Comment:
I think leaflet should be kept as a peerDependency + devDependency.
This way, the consuming app will be able to determine what leaflet version they use.